src/control/cmd/daos_agent/start.go

```diff
@@ -166,6 +156,9 @@ func (cmd *startCmd) Execute(_ []string) error {
 			case syscall.SIGUSR1:
 				cmd.Infof("Signal received.  Caught %s; flushing open pool handles", sig)
 				procmon.FlushAllHandles(ctx)
+			case syscall.SIGUSR2:
```

Comment:
This management of this POSIX signal should be probably documented somewhere such as in the new section of the daos_agent.yml file.

Reply:
@Michael-Hennecke could probably have some good advices for the better location.
However, the admin guide seems a good location for me.